jpa-idempotent-repository/pom.xml

```diff
@@ -0,0 +1,396 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Licensed to the Apache Software Foundation (ASF) under one or more
+    contributor license agreements.  See the NOTICE file distributed with
+    this work for additional information regarding copyright ownership.
+    The ASF licenses this file to You under the Apache License, Version 2.0
+    (the "License"); you may not use this file except in compliance with
+    the License.  You may obtain a copy of the License at
+
+         http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<artifactId>camel-quarkus-examples-jpa-idempotent-repository</artifactId>
+	<groupId>org.apache.camel.quarkus.examples</groupId>
+	<version>3.5.0-SNAPSHOT</version>
+
+	<name>Camel Quarkus :: Examples :: JPA Idempotent Repository</name>
+	<description>Camel Quarkus Example :: JPA Idempotent Repository</description>
+
+	<properties>
+		<quarkus.platform.version>3.4.1</quarkus.platform.version>
+		<camel-quarkus.platform.version>3.5.0-SNAPSHOT</camel-quarkus.platform.version>
```

Comment:
Set this version here similarly to other example in the camel-quarkus-main branch. It's not working locally.

Reply:
ok, got it.